rfc-parser/build.gradle

```diff
@@ -1,9 +1,9 @@
 dependencies {
-    implementation group: 'org.jetbrains', name: 'annotations', version: '16.0.1'
-    implementation group: 'commons-io', name: 'commons-io', version: '2.6'
-    implementation group: 'com.google.guava', name: 'guava', version: '21.0'
-    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+    implementation group: 'org.jetbrains', name: 'annotations', version: annotationsVersion
+    implementation group: 'commons-io', name: 'commons-io', version: commonsIoVersion
+    implementation group: 'com.google.guava', name: 'guava', version: guavaVersion
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: jupiterApiVersion
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: jupiterEngineVersion
```

Comment:
I think this should be declared as "testImplementation" instead of "implementation"